### DISCUSSION TITLE
Virtual Switch to trigger a motion detection

### DISCUSSION THREAD

**Discussion by CWiedermann** [dgreif/ring#938] [Ideas]
First of all, thank you for the amazing plugin!! 
I am very new to homebridge so please forgive me if I am just missing some obvious way this could be done.

In older versions of iOS, when a motion sensor was triggered that was in the same room as a Homekit enabled camera, it was possible to get notifications that contained a still image of the camera video.
Lately, this only works when the motion senor is part of the camera, but that only activates when there is actual motion.

I don´t need it to react to motion though, but have it be triggered by an automation.

It would be awesome if the ring plugin could show a virtual switch that can be triggered by any automation and would have homekit act like a real motion was detected by the camera.

Would you consider this a useful addition, or am I the only one that tries to get manally triggered notifications that contain a still image?

**Comment by dgreif** [comment]
Thanks for the detailed description @CWiedermann.  I don't think this would have a wide user base, and isn't something I'm interested in adding/maintaining.  You are welcome to fork the plugin and add it for your own use tho.
